packages/gatsby/src/commands/develop.ts

```diff
@@ -92,19 +92,19 @@ class ControllableScript {
     this.debugInfo = debugInfo
   }
   start(): void {
+    const args = []
     const tmpFileName = tmp.tmpNameSync({
       tmpdir: path.join(process.cwd(), `.cache`),
     })
     fs.outputFileSync(tmpFileName, this.script)
     this.isRunning = true
-    const args = [tmpFileName]
```

Comment:
we are not passing `tmpFileName` anymore here - can we revert that part or add `args.push(tmpFileName)`?

Reply:
to clarify - now we run `node [...extraArgs]` but we should run `node ${tmpFileName} [...extraArgs]`

Reply:
Fixed, did a bad split of pr